pkg/proxy/winkernel/proxier_test.go

```diff
@@ -765,6 +769,11 @@ func TestCreateDsrLoadBalancer(t *testing.T) {
 		if svcInfo.localTrafficDSR != true {
 			t.Errorf("Failed to create DSR loadbalancer with local traffic policy")
 		}
+		if len(svcInfo.loadBalancerIngressIPs) == 0 {
```

Comment:
This test condition fails. Looking at the structs it's not clear what should actually be compared here or what the expectation is.

Reply:
The fake needs to be changed a lit.